### PR TITLE
Created Hero Section

### PR DESCRIPTION
**Pull request by gsmalik030**
In this hero-branch:
I created a heading, paragraph, and Social media section.
Completed the hero section by arranging all components and adding a background image.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,17 +6,32 @@
...
   <body>
-    <header class="header-container">
-      <nav class="logo">My Logo</nav>
-      <nav class="nav-bar">
-        <img src="images/Union.png" alt="" >
-      </nav>
-    </header>
+    <div class="container">
+      <header class="header-container">
+        <nav class="logo">My Logo</nav>
```
Comment (tufoinnkuo10) [inline index.html:17]: - [ ] Great job. For semantic purposes, only the hamburger should be in the nav tag.
- [ ] For semantic reasons, kindly add a link placeholder to your logo.
eg
`<a href="url">link text</a>`

File: index.html [hunk 2]
```diff
@@ -6,17 +6,32 @@
...
+        <h1>I’m Malik Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <article>
+            <h3>LET’S CONNECT</h3>
+            <img src="images/social media icons.png" alt="">
```
Comment (tufoinnkuo10) [inline index.html:32]: - [ ] please this should not be an image but different social icons. they will help a user connect with you.
- [ ] kindly ensure The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

File: index.html [hunk 3]
```diff
@@ -6,17 +6,32 @@
...
-      <nav class="logo">My Logo</nav>
-      <nav class="nav-bar">
-        <img src="images/Union.png" alt="" >
-      </nav>
-    </header>
+    <div class="container">
+      <header class="header-container">
+        <nav class="logo">My Logo</nav>
+        <nav class="nav-bar">
+          <img src="images/Union.png" alt="" />
```
Comment (tufoinnkuo10) [inline index.html:19]: - [ ] please increase the margin righ of the hamburger icon so it fits the original design.

**Review by tufoinnkuo10 — COMMENTED** [review]
Hi @gsmalik030 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
- [ optional] please avoid merging your PR when it has not yet been approved by a code reviewer 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (gsmalik030) [reply index.html:17]: Thanks for highlighting

**Pull request merged**